ui/redux/actions/publish.js

```diff
@@ -558,7 +561,10 @@ export const doPrepareEdit = (claim: StreamClaim, uri: string, claimType: string
     tags,
   } = value;
 
+  const state = getState();
+  const myClaimForUri = selectMyClaimForUri(state);
   const publishData: UpdatePublishFormData = {
+    claim_id: myClaimForUri || {},
```

Comment:
`claim_id` feels like it should be a string, but we are passing the entire claim object?
I didn't try to confirm it, though.

Reply:
For the null case

Reply:
You're correct. Adjusted.